### DISCUSSION TITLE
Can you opt out of edge to edge?

### DISCUSSION THREAD

**Discussion by jordmccord** [jpudysz/react-native-unistyles#283] [Q&A] [answered]
Hi, love the library and we've started to use it in our native component library to style and theme components.
I've recently implemented the library into our native apps, and I'm having some issues and having to write some hacks to get around the forced edge-to-edge behaviour. Is there any way to opt out of this feature? It is causing headaches hacking around legacy screens having to add the inset into some pages and remove the inset on others, it's also cutting the log error messages from the bottom.

**Comment by jpudysz** [comment, marked as answer]
No, it’s not possible. I’m using it to get updates about insets, and it’s the only reliable method for Android.

Additionally, starting from Android 15, it will be automatically enabled by the OS. This approach also matches how iOS works, so it’s not that hard to adapt.

Regarding the log box, I plan to open a PR in React Native to fix it, but right now, I’m focused on Unistyles 3.0.
